service/src/main/java/uk/nhs/adaptors/oneoneone/cda/report/mapper/CarePlanMapper.java

```diff
@@ -80,6 +84,34 @@ public CarePlan createCarePlanFromSection(POCDMT000002UK01Section cpSection, Enc
             carePlan.setDescription(cpTextContent);
         }
 
+        if (!questionnaireResponseList.isEmpty()) {
+            List<Reference> supportingInformationList = new ArrayList<>();
+            for (QuestionnaireResponse questionnaireResponse : questionnaireResponseList) {
+                supportingInformationList.add(new Reference(questionnaireResponse));
+            }
+            carePlan.setSupportingInfo(supportingInformationList);
+        }
+
+        if (encounter.hasDiagnosis()) {
```

Comment:
Can we pass Condition in a different way here? Diagnosis element will be probably deleted

Reply:
Diagnosis is the only mapper that uses condition. If we are removing it we can call the condition mapper in careplan